platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusUtils.java

```diff
@@ -72,9 +74,10 @@ public static boolean coin(@NonNull final EventImpl event) {
         // find whitening for round
         for (final EventImpl w : judges) { // calculate the whitening byte array
             if (w != null) {
-                final int mn = Math.min(whitening.length, w.getSignature().length);
+                final Bytes sig = w.getBaseEvent().getSignature();
+                final int mn = Math.min(whitening.length, (int) sig.length());
```

Comment:
Do we know why the protobuff model uses long for length? If it is always int maybe we should change protobuff.

Reply:
I have no idea. Maybe @anthony-swirldslabs knows

Reply:
There's not a specific answer.

From theoretical perspective, the protobuf as a protocol doesn't really need to have a limitation on the sizes/lengths. And hence, choosing long as the "longest" integer value for sizes/lengths makes sense.

From practical perspective, the spec seems to impose a hard limit of 2GB on protobuf messages, meaning that it's technically impossible to encode repeated or size-limited fields longer than 2G, which means that a 32-bit integer should suffice for size values.

From yet another perspective, some PBJ APIs were inspired by stream-related APIs where there's not really a limit on the length of the stream. So similar to the point 1 above, using a long makes sense.

I think that the points 1 and 3 played a role historically, and so we ended up with longs.

You may open a feature request against PBJ to try and switch to ints, but I'm unsure about its prioritization, or if there's any other legitimate reasons to keep using longs.

Reply:
Adding @jasperpotts for discussion. I seen the casting to `int` at several places.